harmonizer-cli/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "harmonizer-cli"
```

Comment:
This is bikeshedding at its finest, but how would you feel about calling this `harmonizer-bin` to distinguish it from a full-blown "cli tool"?

I know that I named it `harmonizer-cli` to begin with, but while working in the repo, I often found myself running `cargo run --bin harmonizer-cli` and honestly `cargo run --bin harmonizer-bin` rolls off the fingers a bit easier. Also, the aforementioned distinction between something that's a full-on CLI and something that's just a little binary.

Reply:
Sure! Fixed in a4fc81f0